### PR TITLE
Application voting interface

### LINKED ISSUE #6: Add vote next  application
Add views for users in HackUPC team vote applications.

This votes should be stored to order applications and decided either which ones should be accepted and which ones should be rejected.

- Next application to vote should be the less globally voted (and older if equals) that the user hasn't yet voted
- One user can't vote an application twice
- If a user has voted all applications he should a get a cookie. Like a real one

### PR DESCRIPTION
**Pull request by casassg**
Interface for non-admin users to vote. Similar to the previously designed. Closes #6 

Shippable for now. Additional steps to be followed:

- [x] Add navbar
  - [x] Logout link
  - [x] Change password link

### CODE REVIEW

**Comment by casassg** [comment]
Preview: 
voting-interface


Not sure about the button: "Not sure" (ah the irony) -> Behaves skipping the application

File: register/templates/vote.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            {% if app %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        {# We set the title as lennyface, in out efforts to anonimize the data.. Okay, maybe no #}
+                        <h4 class="text-center">{{ app.lennyface }}</h4>
+                    </div>
+                    <div class="panel-body">
+                        <h4 class="text-center">Personal</h4>
+                        <dl class="dl-horizontal">
+                            {% include 'include/field.html' with desc='Name' value=app.name|add:', '|add:app.lastname %}
```
Comment (ani300) [inline register/templates/vote.html]: The format here should be either "LastName, Name" or "Name LastName" but not "Name, LastName"

File: register/views.py [hunk 2]
```diff
@@ -12,6 +19,51 @@ def get(self, request):
...
+            .annotate(count=Count('vote')) \
+            .order_by('count', 'submission_date') \
+            .first()
+
+    def post(self, request, *args, **kwargs):
+
+        vote_type = models.VOTE_SKIP
+        if request.POST.get('accept'):
+            vote_type = models.VOTE_POSITIVE
+        if request.POST.get('decline'):
```
Comment (ani300) [inline register/views.py]: For the sake of clarity, this should be an else if and not simply an if

File: register/templates/vote.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                            {% if app.devpost and not app.devpost == 'http://' %}
+                                <a target="_blank" href="{{ app.devpost }}" class="icon">
+                                    <img class="icon" src="{% static 'img/devpost.svg' %}"/>
+                                </a>
+                            {% endif %}
+                        </p>
+
+                    </div>
+                    <div class="panel-footer">
+                        <form action="" method="post">
```
Comment (ani300) [inline register/templates/vote.html:62]: Three forms... why?

**Review by ani300 — CHANGES_REQUESTED** [review]
There are three comments, one on format, one on functionality and one on code clarity

**Review by ani300 — APPROVED** [review]

**Comment by casassg** [comment]
What do you think about the not sure button? Keep or remove? @ani300

**Comment by ani300** [comment]
@casassg If we maintain it, change to "Skip this application". I find it useful if not abused, maybe limit the number of skips per user (so no one can skip more than 10 applicants or so)?

**Comment by casassg** [comment]
We could put a limit, that's true. Not easy but not hard either.

**Comment by ani300** [comment]
The other option is to just trust people to do their jobs right.
Did I just say _trust people_? What am I becoming?

**Comment by casassg** [comment]
@ani300 trust the people!! For the moment we will trust people on this, it's an internal tool, so we will explain that skip is similar than negative.

**Pull request merged**